src/packet/secret_key.js

```diff
@@ -493,6 +506,10 @@ class SecretKeyPacket extends PublicKeyPacket {
       throw new Error('Key is not decrypted');
     }
 
+    if (this.skipValidation) {
```

Comment:
Might want a comment here as well about why it's safe to skip, e.g.
```suggestion
    if (this.integrityChecked) {
      // The public key's integrity was already verified, based on the AEAD encryption mechanism.
```